staging/src/k8s.io/kubectl/pkg/describe/describe.go

```diff
@@ -2309,16 +2309,7 @@ func (d *CronJobDescriber) Describe(namespace, name string, describerSettings De
 		}
 		return describeCronJob(cronJob, events)
 	}
-
-	// TODO: drop this condition when beta disappears in 1.25
```

Comment:
I'm not sure if this TODO comment is correct.

[kubectl needs to support +/- 1 minor version skew](https://kubernetes.io/releases/version-skew-policy/#kubectl), so this means kubectl 1.25 should support its use with server 1.24, which will still have batch/v1beta1, unless I misunderstood the deprecation timing.

So I think we need to keep this code until kubectl version 1.26, at a minimum.

Reply:
Thanks @brianpursley You are right. I will hold this pr until 1.26